src/main/webapp/consentGroupContainer/ConsentGroupContainer.js

```diff
@@ -0,0 +1,191 @@
+                    consentKey: this.props.consentKey,
+                    consentGroupUrl: this.props.consentGroupUrl,
+                    approveConsentGroupUrl: this.props.approveConsentGroupUrl,
+                    isAdminUrl: this.props.isAdminUrl,
+                    isViewer: this.props.isViewer,
+                    sampleSearchUrl: this.props.sampleSearchUrl,
+                    rejectConsentUrl: this.props.rejectConsentUrl,
+                    updateConsentUrl: this.props.updateConsentUrl,
+                    projectKey: this.props.projectKey,
+                    serverURL: this.props.serverURL,
+                    discardReviewUrl: this.props.discardReviewUrl,
+                    consentNamesSearchURL: this.props.consentNamesSearchURL,
+                    clarificationUrl: this.props.clarificationUrl,
+                    loadingImage: this.props.loadingImage,
+                    initStatusBoxInfo: this.props.initStatusBoxInfo,
+                    updateDetailsStatus: this.updateDetailsStatus,
+                    updateContent: this.updateContent
+                  })
+                ]),
+              div({
+                key: "documents",
+                title: "Documents",
+              }, [
+                  h(ConsentGroupDocuments, {
+                    attachmentsUrl: this.props.attachmentsUrl,
+                    serverURL: this.props.serverURL,
+                    attachDocumentsUrl: this.props.attachDocumentsUrl,
+                    projectKey: this.props.consentKey,
+                    approveDocumentUrl: this.props.approveDocumentUrl,
+                    rejectDocumentUrl: this.props.rejectDocumentUrl,
+                    sessionUserUrl: this.props.sessionUserUrl,
+                    downloadDocumentUrl: this.props.downloadDocumentUrl,
+                    emailDulUrl: this.props.emailDulUrl,
+                    loadingImage: this.props.loadingImage,
+                    useRestrictionUrl: this.props.useRestrictionUrl,
+                    createRestrictionUrl: this.props.createRestrictionUrl,
+                    removeDocumentUrl: this.props.removeDocumentUrl,
+                    updateDocumentsStatus: this.updateDocumentsStatus
+                  })
+                ]),
+              div({
+                key: "comments",
+                title: "Messages",
+              }, [
+                  h(Fragment, {}, [Comments({
+                    projectKey: this.props.projectKey,
+                    serverURL: this.props.serverURL,
+                    commentsContent: this.state.commentsContent
+                  }
+                  )]),
+                ]),
+              div({
+                key: "history",
+                title: "History",
+              }, [
+                  h(Fragment, {}, [History({
+                    projectKey: this.props.projectKey,
+                    serverURL: this.props.serverURL,
+                    historyContent: this.state.historyContent
```

Comment:
Similar comments here ... `serverURL`, at a minimum, should never have to be a passed in property. That should always be a constant.